sdk/main/src/ED25519PrivateKey.cc

```diff
@@ -52,14 +53,21 @@ const std::vector<std::byte> SLIP10_SEED = { std::byte('e'), std::byte('d'), std
  */
 [[nodiscard]] internal::OpenSSLUtils::EVP_PKEY bytesToPKEY(std::vector<std::byte> bytes)
 {
+  std::vector<std::byte> buildBytes = bytes;
```

Comment:
Any particular reason you wanted to use `buildBytes` instead of `bytes`? I think you could just use `bytes` in this function and not have another copy of the data.

Reply:
Yes it could be, my bad... Copy paste error.